src/languages/handlebars.js

```diff
@@ -5,32 +5,58 @@ Author: Robin Ward <robin.ward@gmail.com>
 Description: Matcher for Handlebars as well as EmberJS additions.
 Category: template
 */
-
 function(hljs) {
   var BUILT_INS = {'builtin-name': 'each in with if else unless bindattr action collection debugger log outlet template unbound view yield'};
+  function MUSTACHE_CONTENTS() {
+    return  {
+      className: 'name',
+      begin: /[a-zA-Z\.-]+/,
+      keywords: BUILT_INS,
```

Comment:
Do any of the tests include a built-in?  Might that be a good idea?

Reply:
done